src/main/webapp/app/shared/subject/subject-dialog.component.html

```diff
@@ -53,7 +53,8 @@
             <select class="form-control"
                     id="field_group_1"
                     name="delusion1"
-                    [(ngModel)]="delusion1$"
+                    [(ngModel)]="delusion1.label"
```

Comment:
the ngmodel needs to be "key" equivalent to what is put in the <option [ngValue]="g.key"